pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -176,7 +176,13 @@ func (b *ServiceBinder) Unbind() (reconcile.Result, error) {
 func updateServiceBindingRequestStatus(
 	dynClient dynamic.Interface,
 	sbr *v1alpha1.ServiceBindingRequest,
+	condition ...conditionsv1.Condition,
```

Comment:
Really really small nitpicking: `conditions` instead of `condition`.

Reply:
Resolved 8383b4f